locale/nl.po

```diff
@@ -8028,21 +7980,18 @@ msgstr "Windows/DOS OEM Cyrillic (CP 866)"
 
 #. TRANSLATORS: Name of keyboard key
 #: ../src/common/accelcmn.cpp:109
-#, fuzzy
 msgid "Windows_Left"
-msgstr "Windows 7"
+msgstr "Windows_Left"
```

Comment:
I don't think all of the keys need translating (though I suppose people that exclusively use Dutch on their systems would disagree) but maybe at least this one, and Windows_Right, could be? In this case "Linker Windows-toets".